--- Running networkx.to_pandas_edgelist#15YY/to_pandas_edgelistYY, strategy: relancer
[ERR MSG]: OrderedDict([('project', 'networkx.to_pandas_edgelist#15YY'), ('notebook', 'to_pandas_edgelistYY'), ('text', '')])
[INFO] This case is fully fixed!


real	0m0.642s
user	0m1.611s
sys	0m6.042s
